src/main/java/seedu/address/model/ContactList.java

```diff
@@ -48,13 +62,45 @@ public void setCourseMates(List<CourseMate> courseMates) {
         this.courseMates.setCourseMates(courseMates);
     }
 
+    /**
+     * Replaces the contents of the group list with {@code groups}.
+     * {@code groups} must not contain duplicate groups.
+     */
+    public void setGroups(List<Group> groups) {
+        this.groups.setGroups(groups);
+    }
+
     /**
      * Resets the existing data of this {@code ContactList} with {@code newData}.
      */
     public void resetData(ReadOnlyContactList newData) {
         requireNonNull(newData);
 
         setCourseMates(newData.getCourseMateList());
+
+        // todo To be implemented
```

Comment:
Use `// TODO: the task to be done` so that major IDEs can recognize this as an "issue".

Reply:
Ok got it